test/test.googleauth.ts

```diff
@@ -35,61 +37,35 @@ afterEach(() => {
   process.env = envCache;
 });
 
-function createIsGCENock(isGCE = true) {
-  nock('http://metadata.google.internal').get('/').reply(200, null, {
-    'metadata-flavor': 'Google'
-  });
+const host = gcpMetadata.HOST_ADDRESS;
+const instancePath = '/computeMetadata/v1beta1/instance';
```

Comment:
What's going on with the `v1` vs `v1beta1`? That needs to be sorted out before this lands (hopefully with `v1` being used uniformly everywhere). Alternative, can't you use the/an export from `gcp-metadata`?